versions.txt

```diff
@@ -4,7 +4,7 @@ json_glib_version=1.2.2
 check_version=0.10.0
 docker_engine_fedora_version=1.12.1
 docker_engine_ubuntu_version=1.12.1-0~xenial
-clear_vm_image_version=15900
+clear_vm_image_version=16020
 clear_vm_kernel_version=4.9.32-71
```

Comment:
we need to sync out also the kernel there when we updated one or the other, so we need the linux-container kernel version that was released in clear image 16020